pkg/manager/manager.go

```diff
@@ -28,8 +28,9 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/healthz"
 
+	"github.com/kubermatic/utils/pkg/util"
```

Comment:
@nmiculinic How do we want to sort our import? How about we put kubermatic related import to a dedicated import group?

Reply:
They are like that, aren't they? with `goimports -w github.com/kubermatic` ?

Reply:
yeah, you are right